src/test/kotlin/com/atlassian/performance/tools/report/api/result/CohortResultTest.kt

```diff
@@ -57,4 +58,41 @@ class CohortResultTest {
         ).build()
         assertThat(actualLatest, not(equalTo(straggler)))
     }
+
+    @Test
+    fun shouldNotBeAffectedByOutliers() {
+        val anticipatedLoad = VirtualUserLoad(
+            virtualUsers = 20,
+            flat = Duration.ofMinutes(10)
+        )
+        val criteria = PerformanceCriteria(
+            mapOf(
+                CREATE_ISSUE_SUBMIT to Criteria(
+                    toleranceRatio = 0.04F,
+                    minimumSampleSize = 12,
+                    acceptableErrorCount = Int.MAX_VALUE
+                )
+            ),
+            virtualUserLoad = anticipatedLoad,
+            maxVirtualUsersImbalance = 18,
+            nodes = 2
+        )
+        val resultsPath = Paths.get("CREATE-ISSUE-WITH-OUTLIERS")
+        val workspace = TestWorkspace(Files.createTempDirectory("cr-workspace"))
+        val rawAlphaResults = LocalRealResult(resultsPath.resolve("alpha")).loadRaw()
+        val rawBetaResults = LocalRealResult(resultsPath.resolve("beta")).loadRaw()
+        val timeline: Timeline = FullTimeline()
+        val edibleAlphaResults = rawAlphaResults.prepareForJudgement(timeline)
+        val edibleBetaResults = rawBetaResults.prepareForJudgement(timeline)
+        val alphaStats = edibleAlphaResults.stats
+        val betaStats = edibleBetaResults.stats
+
+        val verdict = BaselineComparingJudge().judge(
```

Comment:
I think we should write a proper test for `OutlierTrimming` class only. This test tests too much. Such a test would be just a couple lines of code and I believe it would be easier to maintain and fix in the future in case it fails.